preseed.cfg.mcr.sd

```diff
@@ -0,0 +1,47 @@
+d-i debian-installer/locale string en_US
+d-i debian-installer/language string en
+d-i debian-installer/country string US
+
+# Skip creation of a normal user account.
+d-i passwd/make-user boolean false
+d-i passwd/root-login boolean true
+# printf "debian" | mkpasswd -s -m sha-512
+d-i passwd/root-password-crypted password $6$H/WJeEJc$0HnpUXUtjPR/RMpD3qxvb.OGJgTY425jnZn6a9X0YrhGXyEifkR5kTJ20zpv9etzI0k.a9j2G4jMUZjx1XCIH0
+d-i user-setup/allow-password-weak boolean true
+d-i netcfg/choose_interface select auto
+d-i netcfg/get_hostname string unassigned-hostname
+d-i netcfg/get_domain string unassigned-domain
+d-i mirror/country string manual
+d-i mirror/http/hostname string ftp.pl.debian.org
+d-i mirror/http/directory string /debian
+d-i mirror/http/proxy string
+d-i clock-setup/utc boolean true
+d-i clock-setup/ntp boolean true
+d-i time/zone string Europe/Warsaw
+d-i partman/mount_style select uuid
+d-i partman/confirm boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm_nooverwrite boolean true
+d-i partman-auto/disk string /dev/disk/by-id/usb-Multiple_Card_Reader_058F63666485-0:0
```

Comment:
Is this multiple card reader the same for apu1 and apu5?

Reply:
Checked on apu5: 
```
root@debian:~# ls -l /dev/disk/by-id/
total 0
lrwxrwxrwx 1 root root  9 Jun 18 16:51 usb-Multiple_Card_Reader_058F63666485-0:0 -> ../../sdc
```
It's the same